config/types/schema.go

```diff
@@ -151,7 +149,7 @@ type PasswdGroup struct {
 type PasswdUser struct {
 	Create            *Usercreate        `json:"create,omitempty"`
 	Gecos             string             `json:"gecos,omitempty"`
-	Groups            []Group            `json:"groups,omitempty"`
```

Comment:
Won't this be blown away the next time someone runs `generate` from the `ignition.json` schema or is that no longer the case? Also, this rename doesn't really seem needed for this change.